pkg/apis/core/validation/events.go

```diff
@@ -144,9 +144,9 @@ func legacyValidateEvent(event *core.Event) field.ErrorList {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("involvedObject", "namespace"), event.InvolvedObject.Namespace, "does not match event.namespace"))
 		}
 		if len(event.ReportingController) == 0 {
-			allErrs = append(allErrs, field.Required(field.NewPath("reportingController"), ""))
```

Comment:
the field name used should depend on the request version (which is plumbed to `ValidateEventUpdate` as `requestVersion`)... if this is via core/v1, use reportingComponent, if it's via events.k8s.io, use reportingController

Reply:
Got it and already updated, would you please take a review please? thanks